modular_skyrat/modules/shotgunrebalance/code/uplink.dm

```diff
@@ -18,18 +18,18 @@
 	desc = "An additional 8-round slug magazine for use with the Bulldog shotgun. \
 			Who cares about armor when you have flechettes."
 	item = /obj/item/ammo_box/magazine/m12g/flechette
-	purchasable_from = ALL
+	purchasable_from = UPLINK_NUKE_OPS
```

Comment:
delete this file since the whole point of it was making it purchasable from other than nukeops

Reply:
while the reasoning you give here is completely incomprehensible these do now lack a reason to exist